docs/getting_started/aws.rst

```diff
@@ -159,44 +177,59 @@ AMI id that is available in the ``region`` your specified.
 This value will be dependent on the ``source_ami`` that you use. Common values
 are ``centos`` or ``ec2-user``.
 
-``datacenter`` is a name to identify your datacenter, this is important if you have more than one datacenter. 
+``datacenter`` is a name to identify your datacenter, this is important if you have more than one datacenter.
 
-``short_name`` is appended to the name tag and dns (if used) of each of the nodes to help better identify them. 
-
-``control_count``, ``edge_count`` and ``worker_count`` are the number of EC2 instances that will get deployed for each node type. 
+``control_count``, ``edge_count`` and ``worker_count`` are the number of EC2 instances that will get deployed for each node type.
 
 ``control_type``, ``edge_type`` and ``worker_type`` are used to specify the `EC2 instance type <https://aws.amazon.com/ec2/instance-types/>`_
 for your control nodes and worker nodes and they must be compatible with the
 ``source_ami`` you have specified. The default EC2 instance type is an m3.medium.
 
+Security Setup
```

Comment:
So, it is documented in a lot of depth in the 'Security Setup' section, but it blocks terraform from working and asking users to jump back and forth between lots of different pages is a little nasty from a UX/Quickly getting started point of view. Happy to add 'for more info see X', but in terms of deployment flow, it felt needed.

(By the way, have a room of 10 people all actively trying to deploy mantl at once here, this is where a lot of the doc issues/changes i'm submitting are coming from. Anything that catches a few people out or is hard to read/understand i'm treating like a bug)